GOVERNANCE.md

```diff
@@ -20,60 +21,113 @@ Users activities include (but are not limited to):
+Contributors are users contributing in concrete ways to the project. They are one-time participation or are new contributors to the project. Contribution can take place in many ways:
 
 - Supporting new users via the [user discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine)
-- Submitting patches to fix bugs or add features
+- Submitting patches to fix bugs or add features via pull request
 - Identifying requirements
 - Providing graphics and web design
-- Writing and maintaining the [documentation](https://github.com/OpenRefine/OpenRefine/wiki)
+- Writing and maintaining the [documentation](https://github.com/OpenRefine/OpenRefine/tree/master/docs)
+
+Contributors abide by the project’s [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md)
 
-How to become an OpenRefine contributors? The [documentation for developers](https://github.com/OpenRefine/OpenRefine/wiki/Documentation-For-Developers) is a good place to start getting familiar with the code base.  We invite contributors to share their feature development and patch ideas through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open). It is recommended to start with small patches and share your code early so the community can provide feedback and guidance.
+How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)
 
-### Committers
-Committers are contributors who have shown dedication to OpenRefine, have a deep understanding of the code base and project strategy and work well with contributors and users. Committers have no more authorities than contributors, and they should engage with the community through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open) regarding their intention. However, committers have earned enough trust from the community to have direct access to the project code base without having to submit pull request. Committers seek approval after the contribution is made, rather than before.
- 
-Therefore committers:
+### Comitters 
+If you make regular contribution to OpenRefine, you will most likely become a comitter. 
+
+Comitters have no more authorities than contributors, and they should engage with the community through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open) regarding their intention. However, Comitters have earned enough trust from the community to have direct access to the project code base without having to submit pull request. Comitter seek approval after the contribution is made, rather than before.
```

Comment:
We should use the terms **Project Admins**, **Project Owners** and **Contributors**.
But I caution on even adding Project Owners into this Governance because we want Contributors to actually feel like Owners, being responsible for the health of the project and taking an active role and continuing in its development.  So, to that end, I really think we can simplify with only 2 levels...**Project Admins**, **Contributors** ?

Reply:
From @tfmorris  comment #3198#issuecomment-721289810, and in today's call I understood we agreed to contributors & committers.   I really like to close this PR and move to the next steps. 
If we can't get an agreement on the terminology by next Monday I ask for a vote on the dev mailing list (using a survey).

Reply:
I am OK with the current terminology.